R/tm_g_ae_oview.R

```diff
@@ -291,19 +286,22 @@ srv_g_ae_oview <- function(id,
 
       teal.code::chunks_reset(envir = environment())
 
-      validate(need(nlevels(ANL[[input$arm_var]]) > 1, "Arm needs to have at least 2 levels"))
       validate_has_data(ANL, min_nrow = 10)
+
+      iv_comp <- shinyvalidate::InputValidator$new()
+      iv_comp$add_rule("arm_trt", comp_arm, comparison = input$arm_ref)
+      iv_comp$add_rule("arm_ref", comp_arm, comparison = input$arm_trt)
```

Comment:
we can also use in-house functions here:
```
iv_comp$add_rule("arm_trt", shinyvalidate::sv_not_equal(input$arm_ref, message_fmt = "Must not be equal to Control"))
      iv_comp$add_rule("arm_ref", shinyvalidate::sv_not_equal(input$arm_trt, message_fmt = "Must not be equal to treatment"))
```

Reply:
done